src/main/java/training/supportbank/PrinterFormatter.java

```diff
@@ -0,0 +1,25 @@
+package training.supportbank;
+
+import java.math.BigDecimal;
+import java.util.*;
+
+public class PrinterFormatter {
+
+    public static String convertToCurrency(BigDecimal value, Locale locale) {
+        String outputValue;
+        String outputCurrency = Currency.getInstance(locale).getSymbol(locale);
+        outputValue = outputCurrency + value.abs().toString();
+        return outputValue;
+    }
+
+    public static List<String> createScript(Map<String, BigDecimal> calculatedMap) {
```

Comment:
Have a think about the names in this function - `createScript`, `calculatedMap` and `outputArray` are very generic names, but they do something very specific because they know about the concept of owing money. How could you name the method and variables to make the purpose more obvious?

Reply:
outputArray -> currencyValueString (in convertToCurrency method)

createScript -> convertMapToSentence
calculatedMap -> uniqueNameSubtotalMap
outputArray -> uniqueNameSubtotalSentenceArray

Reply:
NB. Use dependency injection for PrinterFormatter into Printer